sqlmesh/core/console.py

```diff
@@ -728,7 +740,7 @@ def _show_summary_tree_for(
             if direct.children:
                 tree.add(direct)
             if indirect.children:
-                tree.add(indirect)
+                tree.add(indirect if self.verbose else self._limit_model_names(indirect))
```

Comment:
should we just push down the verbose check in `_limit_model_name`? So it's only done in one place.

Reply:
Refactored the function to contain the check.